evals/baselines/ollama-gemma4-latest.NOTES.md

```
@@ -0,0 +1,62 @@
+curl -s http://localhost:11434/api/tags \
+  | jq '.models[] | select(.name=="gemma4:latest") | .digest'
+# Expect prefix: c6eb396dbd59
+```
+
+If either check disagrees with this file, the `:latest` tag has been
+re-pulled. Don't trust the comparison until you re-bless or re-pin the
+baseline filename to a stable tag.
+
+## Run summary at bless time
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)